### PR TITLE
test: create test setup using the mocked connection (DEV-2842)

### PR DESCRIPTION
**Pull request by BalduinLandolt**

### CODE REVIEW

**Review by Nora-Olivia-Ammann — APPROVED** [review]

File: test/unittests/test_xmlupload/test_upload_stash_with_mock.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+            stash=stash,
+            id2iri_mapping=id2iri_mapping,
+            con=con,
+            verbose=False,
+        )
+        assert nonapplied is None
+
+
+class TestUploadTextValueStashes:
+    def test_upload_link_value_stash(self) -> None:
```
Comment (jnussbaum) [inline test/unittests/test_xmlupload/test_upload_stash_with_mock.py:97]: Not sure about your terminology, but perhaps you mean:

```suggestion
    def test_upload_text_value_stash(self) -> None:
```

File: test/unittests/test_xmlupload/test_upload_stash_with_mock.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+        )
+        nonapplied = _upload_stash(
+            stash=stash,
+            id2iri_mapping=id2iri_mapping,
+            con=con,
+            verbose=False,
+        )
+        assert nonapplied is None
+
+    def test_not_upload_link_value_stash_if_uuid_not_on_value(self) -> None:
```
Comment (jnussbaum) [inline test/unittests/test_xmlupload/test_upload_stash_with_mock.py:142]: Not sure about your terminology, but perhaps you mean:

```suggestion
    def test_not_upload_text_value_stash_if_uuid_not_on_value(self) -> None:
```

**Review by jnussbaum — APPROVED** [review]
LGTM

File: test/unittests/test_xmlupload/test_upload_stash_with_mock.py (hunk 1 above)
Comment (BalduinLandolt) [reply test/unittests/test_xmlupload/test_upload_stash_with_mock.py]: yes, those were copy-paste mistakes

**Pull request merged**